priv/repo/migrations/20240605015345_create_stored_procedure_to_purge_package_searches.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Hexpm.RepoBase.Migrations.CreateStoredProcedureToPurgePackageSearches do
```

Comment:
We don't use stored procedures elsewhere in the repo. Any reason this cannot be an elixir function?

Reply:
That's cool. I think it just depends on how you want to trigger it.
Are there any other functions in the codebase that do something like this? (ie. where do they live?)
Would you want to set up a cron in GCP to run that function periodically?

Reply:
We can use https://github.com/citusdata/pg_cron, that extension is available on Cloud SQL or we can set up a cron in kubernetes that runs a release task https://github.com/hexpm/hexpm-ops/blob/main/kubernetes/prod/hexpm-stats.yml.

Reply:
cool. my first impression is to switch to a release task for visibility and testability. Adding a pg extension seems to be going down the more opaque path.
WDYT?

Reply:
Yep, sounds good